src/serializer/LazyObjectsSerializer.js

```diff
@@ -105,6 +118,121 @@ export class LazyObjectsSerializer extends ResidualHeapSerializer {
     return getOrDefault(this._valueLazyIds, obj, () => this._lazyObjectIdSeed++);
   }
 
+  _isLazyObject(val: Value) {
+    // Currently, only support lazy raw object.
```

Comment:
What does "raw" mean?

Reply:
That maps to ResidualHeapSerializer's serializeValueObject() method. In it, we only support serializing objects that is not created using constructor and is not objects that we do special handling for, like Map, Date, String, Number etc.... I call this category of objects "raw objects" and there is a serializeValueRawObject() method for it. 
Not sure what is a better name for it? Maybe "literal object"?